src/Microsoft.TestPlatform.ObjectModel/Framework.cs

```diff
@@ -53,19 +59,19 @@ public static Framework FromString(string frameworkString)
                 switch (frameworkString.Trim())
                 {
                     case "Framework35":
-                        frameworkName = new FrameworkName(".NETFramework,Version=v3.5");
+                        frameworkName = new FrameworkName(Constants.DotNetFramework35);
                         break;
                     case "Framework40":
-                        frameworkName = new FrameworkName(".NETFramework,Version=v4.0");
+                        frameworkName = new FrameworkName(Constants.DotNetFramework40);
                         break;
                     case "Framework45":
-                        frameworkName = new FrameworkName(".NETFramework,Version=v4.5");
+                        frameworkName = new FrameworkName(Constants.DotNetFramework45);
                         break;
                     case "FrameworkCore10":
-                        frameworkName = new FrameworkName(".NETCoreApp,Version=1.0");
+                        frameworkName = new FrameworkName(Constants.DotNetFrameworkCore10);
                         break;
                     case "FrameworkUap10":
-                        frameworkName = new FrameworkName("Uap,Version=10.0");
+                        frameworkName = new FrameworkName(Constants.DotNetFrameworkUap10);
                         break;
                     default:
                         return null;
```

Comment:
Should we be returning the DefaultFramework here instead of null?

Reply:
Not change behavior in this PR.